### PR TITLE
Python code for replaying historical data

### PR DESCRIPTION
**Pull request by sinakiaei**
- Add keys for AWS
- Add dataset
- Develop code for replaying data

### CODE REVIEW

**Comment by sarasaeedi** [comment]
@sinakiaei To improve your programming skills, please follow the code documentation practices. I couldn't find any description while reviewing your codes!!!

File: replay-historical-data/Pomelo12Tail.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import time
+from datetime import datetime
+import pandas as pd
+import sys
+import AWSIoTPythonSDK
+import os
+from AWSIoTPythonSDK.MQTTLib import AWSIoTMQTTClient
+sys.path.insert(0, os.path.dirname(AWSIoTPythonSDK.__file__))
+
+myMQTTClient = AWSIoTMQTTClient('pomelo')
```
Comment (lkcozy) [inline replay-historical-data/Pomelo12Tail.py:12]: Consider adding a config to include mqtt client name, topic, port, etc. There are different ways to achieve this. You can find more references from https://hackernoon.com/4-ways-to-manage-the-configuration-in-python-4623049e841b and https://martin-thoma.com/configuration-files-in-python/
Anyway, it's totally up to you.

**Review by lkcozy — APPROVED** [review]
Looks good to me. Just have one suggestion regarding the optional configuration.

File: replay-historical-data/readme.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Replay PoMELO historical data
```
Comment (lkcozy) [inline replay-historical-data/readme.md:1]: Consider adding a section to describe how it works briefly?

**Pull request merged**